test/lint/README.md

```
@@ -17,6 +17,42 @@ docker run --rm -v $(pwd):/bitcoin -it bitcoin-linter
+
+To run all the lint checks in the test runner outside the docker you first need
+to install the rust toolchain using your package manager of choice or
+[rustup](https://www.rust-lang.org/tools/install).
+
+Then you can use:
+
+```sh
+( cd ./test/lint/test_runner/ && cargo fmt && cargo clippy && RUST_BACKTRACE=1 cargo run )
+```
+
+#### Dependencies
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)